### DISCUSSION TITLE
Pointers to implementing motion blur for entire stage?

### DISCUSSION THREAD

**Discussion by johot** [pixijs/pixijs#9606] [General]
Thank you for such a wonderful library! 

I am currently trying to add motion blur to the entire stage of a pixi project I am building. I have found the built in motion blur filters but they work with velocity vectors or zoom against a point. I am looking to implement motion blur for video frames or well anything where I can't just motion blur horizontally and vertically (or by zooming) instead I need to create motion blur based on the previous frames.

Have anyone any ideas about how this could be achieved using pixi?

Thank you!

**Comment by paulgrassler** [comment]
Hey, did you ever come up with something? I am currently in a similar situation.
